process.c

```diff
@@ -540,18 +541,21 @@ get_ppid(void)
 
 /*
  *  call-seq:
- *     Process.ppid   -> integer
+ *    Process.ppid -> integer
  *
- *  Returns the process id of the parent of this process. Returns
- *  untrustworthy value on Win32/64. Not available on all platforms.
+ *  Returns the process ID of the parent of the current process:
  *
- *     puts "I am #{Process.pid}"
- *     Process.fork { puts "Dad is #{Process.ppid}" }
+ *    puts "Pid is #{Process.pid}."
+ *    fork { puts "Parent pid is #{Process.ppid}." }
  *
- *  <em>produces:</em>
+ *  Output:
  *
- *     I am 27417
- *     Dad is 27417
+ *    Pid is 271290.
+ *    Parent pid is 271290.
+ *
+ *  Not available on all platforms.
+ *
+ *  Returns an untrustworthy value on Windows.
```

Comment:
The availability is same as `pid`, but it is always 0 on Wasm.
As for Windows, I guess "an untrustworthy value" means that it is 0 on old Windows 9x, but not sure.

```suggestion
 *  Does not return a trustworthy value on all platforms.
```

Reply:
"Does not return a trustworthy value on all platforms." sounds like it never returns a trustworthy value on any platforms. How about "May not return a trustworthy value on certain platforms."

Reply:
Modified (per @peterzhu2118).  Okay with you, @nobu?